dev/com.ibm.ws.transport.http/src/com/ibm/ws/genericbnf/PasswordNullifier.java

```diff
@@ -19,25 +19,39 @@
     /** Search patterns to use for passwords */
     private static final String PASSWORD_PATTERN = "password=";
     private static final String CLIENT_SECRET_PATTERN = "client_secret=";
+    private static final String BASICAUTH_PATTERN = "basic ";
+    private static final String AUTHORIZATION_HEADER = "Authorization";
```

Comment:
We might want to use `com.ibm.wsspi.http.channel.values.HttpHeaderKeys.HDR_AUTHORIZATION` here instead